pages/manageIssues/ManageIssuesHandler.inc.php

```diff
@@ -52,7 +52,21 @@ function index($args, $request) {
 		AppLocale::requireComponents(LOCALE_COMPONENT_APP_EDITOR);
 
 		$templateMgr = TemplateManager::getManager($request);
+		if ($request->getUserVar('json') == 1) {
```

Comment:
Is this a work in progress? I don't see this being called in both modes, at a glance.